integration-cli/docker_utils.go

```diff
@@ -267,7 +267,7 @@ func (d *Daemon) Cmd(name string, arg ...string) (string, error) {
 
 func daemonHost() string {
 	daemonUrlStr := "unix:///var/run/docker.sock"
-	if daemonHostVar := os.Getenv("DOCKER_TEST_HOST"); daemonHostVar != "" {
```

Comment:
https://github.com/docker/docker/blob/99ca215c65e9a482f981790f2d4dbe8d4ccce4be/project/make/.integration-daemon-start#L27

Because `DOCKER_TEST_HOST` was only ever intended to be a knob you pass to the scripts to control the `DOCKER_HOST` value used by Docker, which all these tests invoke (and only some of which actually use this `daemonHost()` value, and so are relying on the line of the script I linked above anyhow).  If those two happened to get out of sync somehow, we'd have a real fun time.